src/sections/Question/Question.module.scss

```diff
@@ -0,0 +1,80 @@
+@use '../../styles/mixins.module';
+@use '../../styles/colors.module';
+
+.content {
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+  grid-gap: 2rem;
+
+  @include mixins.media(laptop) {
+    grid-template-columns: 1fr;
+    grid-gap: 0;
+    justify-items: center;
+
+    & .sectionText {
+      text-align: center;
+    }
+  }
+}
+
+.inputContainer {
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+  grid-gap: 1rem;
+  padding-bottom: 1rem;
+
+  @include mixins.media(laptop) {
+    display: block;
+  }
+}
+
+.message {
+  grid-column: span 2;
+}
+
+form {
+  margin-top: 2.75rem;
+  box-sizing: border-box;
+  align-items: center;
+  border-radius: 0.375rem;
+  border: solid colors.$primary-3 1px;
+  padding: 1rem 2rem;
+
+  @include mixins.media(laptop) {
+    width: 60%;
+  }
+
+  @include mixins.media(mobile) {
+    width: 100%;
+  }
+
+  & button {
+    margin-left: auto;
+  }
+}
+
+.input {
+  margin-bottom: 1rem;
+  align-content: flex-start;
+  gap: 0;
+
+  @include mixins.media(laptop) {
+    margin-bottom: 1.5rem;
+  }
+
+  & input,
+  textarea {
+    margin-top: 0.5rem;
+    margin-bottom: 0.5rem;
+  }
+
+  & textarea {
+    resize: vertical;
+    min-height: 2rem;
+    max-height: 15rem;
+  }
+
+  & span {
+    height: 0;
+  }
```

Comment:
Hmm I believe most of this is also part of the hacky fix mentioned above.
Except for lines 73-74, those were just for restricting the min/max size of the textarea. I can remove it.

Reply:
Hmm okay I see what you mean. I think we can sprinkle some JS so that the gap disappears depending on if the error exist, and combine it with media queries so that in single column it keeps the gap since we do want that negative space between the inputs - even if it means the page shifts